mllib/src/test/scala/org/apache/spark/mllib/linalg/distributed/RowMatrixSuite.scala

```diff
@@ -20,10 +20,8 @@ package org.apache.spark.mllib.linalg.distributed
 import java.util.Arrays
 
 import scala.util.Random
-
-import breeze.linalg.{norm => brzNorm, svd => brzSvd, DenseMatrix => BDM, DenseVector => BDV}
+import breeze.linalg.{DenseMatrix => BDM, DenseVector => BDV, norm => brzNorm, svd => brzSvd}
 import breeze.numerics.abs
-
```

Comment:
I think the style checker wants this blank line

Reply:
fixed